test/e2e/specs/AutolendingPage.spec.js

```diff
@@ -1,7 +1,7 @@
 import { subDays } from 'date-fns';
 import wwwPageMock from '../fixtures/wwwPageMock';
 
-describe('Autolending Page Spec', () => {
+describe.skip('Autolending Page Spec', () => {
```

Comment:
Skipping this test suite for now because it requires authentication. This will be revisited and refactored after support for authentication in testing is added during VUE-721.